src/modules/mount/main.py

```diff
@@ -80,37 +84,44 @@ def mount_partitions(root_mount_point, partitions):
             subprocess.check_call(["umount", "-v", root_mount_point])
 
             if "luksMapperName" in partition:
-                libcalamares.utils.mount("/dev/mapper/{!s}".format(partition["luksMapperName"]),
+                libcalamares.utils.mount("/dev/mapper/"
+                                         + partition["luksMapperName"],
```

Comment:
I wonder (similar to other PR) about changing the explicit conversion to an implicit assumption. A different coding style which pycodestyle accepts (and I would too) is:

```
                    libcalamares.utils.mount(
                        "/dev/mapper/{!s}".format(partition["luksMapperName"]),
                        root_mount_point + "/home",
                        fstype,
                        ",".join(
                            ["subvol=@home", partition.get("options", "")]),
                        )
```